pkg/istio/destinationrule.go

```diff
@@ -64,7 +64,7 @@ func DestinationRuleRevertor(ctx model.SessionContext, ref *model.Ref) error { /
 		}
 
 		ctx.Log.Info("Found DestinationRule", "name", resource.Name)
-		mutatedDr, err := revertDestinationRule(*dr)
+		mutatedDr, err := revertDestinationRule(*dr, ctx)
```

Comment:
Why is revert taking 'ctx' while mutate just the 'name'?

Reply:
Missed that, the code was changed multiple times where I was fiddling with the `ctx` to pass more metadata. That's clearly leftover. Fixed with 593618c